typescript/schedules.md

```diff
@@ -0,0 +1,1257 @@
+# Schedules
+
+- [Docs](https://docs.temporal.io/workflows/#schedules)
+- gRPC API:
+  - [methods](https://github.com/temporalio/api/blob/799926c86eb13d8a9717d3561ab9b0df43796c06/temporal/api/workflowservice/v1/service.proto#L328-L370)
+  - [`request_response.proto`](https://github.com/temporalio/api/blob/799926c86eb13d8a9717d3561ab9b0df43796c06/temporal/api/workflowservice/v1/request_response.proto#L821-L957)
+  - [`schedule/v1/message.proto`](https://github.com/temporalio/api/blob/master/temporal/api/schedule/v1/message.proto)
+  - [`ScheduleOverlapPolicy` enum](https://github.com/temporalio/api/blob/master/temporal/api/enums/v1/schedule.proto)
+
+## TS API
+
+```ts
+import { ScheduleClient, ScheduleOverlapPolicy } from '@temporalio/client'
+import { addWeeks } from 'date-fns'
+import { myWorkflow } from './workflows'
+
+const client = new ScheduleClient();
+
+const schedule = await client.create({
+  id: 'schedule-biz-id',
+  spec: {
+    // every hour at minute 5
+    intervals: [
+      {
+        every: '1h',
+        at: '5m',
+      },
+    ],
+    // every 20 days since epoch at day 2
+    // intervals: [{
+    //   every: '20d',
+    //   at: '2d'
+    // }]
+    skip: [
+      {
+        // skip 11:05 pm
+        hour: 23,
+        minute: 5,
+      },
+    ],
+    endAt: addWeeks(new Date(), 4),
+    jitter: '30s',
+    timezone: 'US/Pacific',
+  },
+  action: {
+    workflowId: 'wf-biz-id',
+    type: myWorkflow,
+    args: ['arg1', 'arg2'],
+  },
+  overlap: ScheduleOverlapPolicy.BufferOne,
+  catchupWindow: '2m',
+  pauseOnFailure: true,
+  note: 'started schedule',
+  pause: true, // this sets `state.paused: true` (default false)
```

Comment:
Use the declarative form instead
```suggestion
  paused: true, // this sets `state.paused: true` (default false)
```

Reply:
Why?

Reply:
> Ideally the list / describe response are as similar as possible to the create request.

Reply:
Didn't understand if you were applying this or not, I saw this resolved.

Reply:
Yep, applied locally as I saw found your reason